src/transformers/models/gpt2/modeling_gpt2.py

```diff
@@ -1479,9 +1479,24 @@ def __init__(self, config):
     @add_start_docstrings_to_model_forward(GPT2_INPUTS_DOCSTRING)
     @add_code_sample_docstrings(
         processor_class=_TOKENIZER_FOR_DOC,
-        checkpoint="microsoft/DialogRPT-updown",
+        checkpoint="brad1141/gpt2-finetuned-comp2",
         output_type=TokenClassifierOutput,
         config_class=_CONFIG_FOR_DOC,
+        expected_loss=0.25,
+        expected_output=[
+            "Lead",
+            "Lead",
+            "Lead",
+            "Position",
+            "Lead",
+            "Lead",
+            "Lead",
+            "Lead",
+            "Lead",
+            "Lead",
+            "Lead",
+            "Lead",
+        ],
```

Comment:
Hi @sgugger 

Patrick delegate the responsibility to me. I am still wondering if you have any extra comment for this PR though.

There is only 2 checkpoints on the Hub for GPT2 + token classification.
This one is trained on [writing document evaluation dataset](https://www.kaggle.com/code/bakar31/writing-evaluation-noob-eda/notebook). The output for this example is therefore not really meaningful. However, I am in favor to merge as it is.

Reply:
@sgugger just following up on this. are we gonna move on this? and close the issue? Thanks =)

Reply:
Sorry I missed my input was required here. Thanks for the pin @ArEnSc ! Fine by me but we should deactivate formatting for that one line to avoid wasting all that vertical space (and have the list back on one line). You can do so with a comment
```
# fmt: off
```
before and
```
# fmt: on
```
after.

Reply:
@sgugger will do!